cmd/activator/main.go

```diff
@@ -294,6 +295,7 @@ func main() {
 	}
 	ah = activatorhandler.NewRequestEventHandler(reqChan, ah)
 	ah = tracing.HTTPSpanMiddleware("handle_request", ah)
+	ah = &ochttp.Handler{Handler: ah}
```

Comment:
Could we make this be part of the tracing.HTTPSpanMiddleware above? My intention was for that to be a general "use this handler to extract traces" - we could probably replace its contents with this.